code/modules/government/medical/medical_bills.dm

```diff
@@ -22,27 +22,63 @@ var/global/list/medical_bills = list()
 	var/can_edit = 1							//	Can the minister of health/president edit this?
 
 /datum/medical_bill/basic_treatment
-	name = "Basic Treatment"
-	description = "A basic treatment which usually involves items within any medical pack."
-	cost = 8
+	name = "Damage (Oxyloss, Toxin, Burns, Brute) under 50 ticks"
+	description = "A basic treatment which usually involves items within any medical pack under 50 ticks."
+	cost = 60
+	insurance_coverage = INSURANCE_BASIC
+	
+/datum/medical_bill/basic_treatment
+	name = "Damage (Oxyloss, Toxin, Burns, Brute) over 50 ticks"
+	description = "A basic treatment which usually involves items within any medical pack over 50 ticks."
+	cost = 80
+	insurance_coverage = INSURANCE_BASIC	
+
+/datum/medical_bill/basic_treatment

```

Comment:
Hm you have /datum/medical_bill/basic_treatment thrice here on line 24, 30, and 36. Name the paths differently or the game will overwrite them.